Content.Shared/Hands/EntitySystems/SharedHandsSystem.cs

```diff
@@ -69,9 +69,9 @@ public virtual void RemoveHand(EntityUid uid, string handName, HandsComponent? h
         if (!handsComp.Hands.Remove(handName, out var hand))
             return;
 
+        handsComp.SortedHands.Remove(hand.Name);
         TryDrop(uid, hand, null, false, true, handsComp);
         hand.Container?.Shutdown();
```

Comment:
Shouldn't the hand be removed after the TryDrop but before the shutdown?

Reply:
I didn't look into it too thoroughly, but I don't think it should cause any issues. This is already being called after the hand got removed hands dictionary, which is what is actually required to get a hand from a hand name.

This change is trying to ensure that hands always get removed simultaneously from the list & dictionary, so that you don't somehow end up with it not getting removed from the list because TryDrop throws an exception.